docs/source/upgrade.rst

```diff
@@ -91,8 +109,16 @@ A new module :mod:`cupyx.profiler` is added to host all profiling related APIs i
 
 The old routines are deprecated.
 
+:func:`cupy.cuda.Device.use` now returns ``self`` to get aligned with :func:`cupy.cuda.Stream.use()`.
```

Comment:
I think non-breaking changes need not be covered here. (otherwise, we have to cover most of the changes 😄 )

Reply:
Okay! I didn't think well that the change does not affect users' existing codes...